src/index.ts

```diff
@@ -72,9 +76,16 @@ export const Rune: RuneExport = {
     }
     const score = Rune._getScore()
     Rune._validateScore(score)
-    globalThis.postRuneEvent?.({ type: "GAME_OVER", score, challengeNumber: Rune.getChallengeNumber() })
+
+    // Reset randomness to be deterministic across plays
+    const challengeNumber = Rune.getChallengeNumber()
+    Rune.deterministicRandom = Rune._randomNumberGenerator(challengeNumber)
+
+    globalThis.postRuneEvent?.({ type: "GAME_OVER", score, challengeNumber })
   },
   getChallengeNumber: () => globalThis._runeChallengeNumber ?? 1,
+  // @ts-ignore This will be set by onLoad()
+  deterministicRandom: undefined,
```

Comment:
There's a test that covers that this gets set correctly.

Reply:
Fixed by @mattbretl with https://github.com/rune/rune-games-sdk/pull/38/commits/46ad431e06aa54e01bda9529f5760279942f8862.